standard_libs/libraries/textdisplayhelper/source.rb

```diff
@@ -30,4 +30,43 @@ def id_ranges_display(items:)
     ranges.map! { |r| r.length == 1 ? r.first.to_s : "#{r.first} - #{r.last}" }
     ranges.to_sentence
   end
+
+  # Creates show block following instructions for show block
+  #
+  # @param title 'String' the string of things to show
+  # @param show_block: 'Array<Hash>' hash to represent each line
+  #        [{display: Array<String>/String/Table, type: 'note','bullet','table'}]
+  def display(title:, show_block:)
```

Comment:
Ben created a back-end method for show blocks that should simplify this pattern quite a bit.